### PR TITLE
updated mediaconnection

### PR DESCRIPTION
**Pull request by matsub**

### CODE REVIEW

File: docs/en/mediaconnection.md [hunk 1]
```diff
@@ -1,144 +1,135 @@
-Class that manages media connections to other peers.
+The `MediaConnection` is a class that manages media connections to another peer.
```
Comment (leader22) [inline docs/en/mediaconnection.md]: > manages media connections

[IMO] The connection is always single...?

File: docs/en/mediaconnection.md [hunk 2]
```diff
@@ -1,144 +1,135 @@
...
+| remoteId | string  | PeerId of the peer this connection connect to.                                                                                                                                                                                                                                                                                       |
+| peer     | string  | **Deprecated** PeerId of the peer this connection connect to. Use `remoteId` instead.                                                                                                                                                                                                                                                |
 
 ## Methods
 
-### answer
+### `answer(stream[, options])`
 
-Create and send an answer message.
+Create and send an answer of the media connection request.
```
Comment (leader22) [inline docs/en/mediaconnection.md]: > answer of the media connection request.

[IMO] answer for media connection **offer** is better?

File: docs/en/mediaconnection.md [hunk 3]
```diff
@@ -1,144 +1,135 @@
...
 
-| Name | Type | Required | Default | Description |
-| --- | --- | --- | --- | --- |
-| videoBandwidth | number | | | A max video bandwidth(kbps) |
-| audioBandwidth | number | | | A max audio bandwidth(kbps) |
-| videoCodec | string | | | A video codec like 'H264' |
-| audioCodec | string | | | A video codec like 'PCMU' |
+| Name                | Type    | Required | Default | Description                                                                                                   |
+|---------------------|---------|----------|---------|---------------------------------------------------------------------------------------------------------------|
+| videoBandwidth      | number  |          |         | Max bandwidth of the video track receiving from the connected peer. (kbps)                                    |
```
Comment (leader22) [inline docs/en/mediaconnection.md]: [SHOULD] These option lists are the same with `Peer`. We should use same expressions.

File: docs/en/mediaconnection.md [hunk 4]
```diff
@@ -1,144 +1,135 @@
...
+receive mode even if the former connection was in receive only mode.
 
 #### Parameters
 
-| Name | Type | Optional | Default | Description |
-| --- | --- | --- | --- | --- |
-| stream | MediaStream | | | The stream to replace the old stream with. |
+| Name   | Type          | Required | Default | Description                          |
+|--------|---------------|----------|---------|--------------------------------------|
+| stream | [MediaStream] | ✔        |         | A MediaStream object to be replaced. |
```
Comment (leader22) [inline docs/en/mediaconnection.md]: > A MediaStream object

[IMO] Just `A MediaStream` is enough and better to distinguish normal object.

**Comment by leader22** [comment]
👀

File: docs/en/mediaconnection.md (hunk 1 above)
Comment (matsub) [reply docs/en/mediaconnection.md]: I didn’t think so much about this :thinking_face:
plz gimme a time to check this.

**Pull request merged**